pkg/kubernetes/scripts/app.js

```diff
@@ -210,8 +209,10 @@
 
             var promise = discoverSettings().then(function(settings) {
                 var ret = [];
-                if (settings.flavor === "openshift")
+                if (settings.flavor === "openshift") {
+                    ret.push(loader.watch("projects", $rootScope));
```

Comment:
Does this require a certain minimal Kubernetes version to work? This watch isn't being used (yet) in our code (unless the `statusPhase("Active")` from the next commit relies on that?), so our tests wouldn't notice if that fails?